cli: filter standalone NOTICE lines in debug zip test output

Abhinav1299 opened the pull request:
Remove intermittent license-related NOTICE messages from zip output in tests.

Resolves: #157888, #157889
Epic: None

Release note: None

cockroach-teamcity commented:
<!-- Reviewable:start -->
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/cockroachdb/cockroach/158033)
<!-- Reviewable:end -->

Abhinav1299 committed:
cli: filter standalone NOTICE lines in debug zip test output

Remove intermittent license-related NOTICE messages from zip output in tests.

Resolves: #157888, #157889
Epic: None

Release note: None

aa-joshi reviewed (approved):
<img class="emoji" title=":lgtm:" alt=":lgtm:" align="absmiddle" src="https://reviewable.io/lgtm.png" height="20" width="61"/>

> *[Reviewable](https://reviewable.io/reviews/cockroachdb/cockroach/158033#-:-OeR81l2BsK34cG5D0fR:bnfp4nl)* status: <span>:shipit:</span> complete! 1 of 0 LGTMs obtained (waiting on @arjunmahishi and @kyle-a-wong)



<!-- Sent from Reviewable.io -->

Abhinav1299 commented:
TFTR!
bors r+
